pkg/operator/ceph/nfs/nfs.go

```diff
@@ -172,7 +173,7 @@ func (r *ReconcileCephNFS) removeServerFromDatabase(nfs *cephv1.CephNFS, name st
 func (r *ReconcileCephNFS) runGaneshaRadosGrace(nfs *cephv1.CephNFS, name, action string) error {
 	nodeID := getNFSNodeID(nfs, name)
 	cmd := ganeshaRadosGraceCmd
-	args := []string{"--pool", nfs.Spec.RADOS.Pool, "--ns", nfs.Spec.RADOS.Namespace, action, nodeID}
+	args := []string{"--cephconf", cephclient.CephConfFilePath(r.context.ConfigDir, nfs.Namespace), "--userid", "rookoperator", "--pool", nfs.Spec.RADOS.Pool, "--ns", nfs.Spec.RADOS.Namespace, action, nodeID}
```

Comment:
```suggestion
	args := []string{"--cephconf", cephclient.CephConfFilePath(r.context.ConfigDir, nfs.Namespace), "--userid", r.clusterInfo.CephCred.Username, "--pool", nfs.Spec.RADOS.Pool, "--ns", nfs.Spec.RADOS.Namespace, action, nodeID}
```

Reply:
`r.clusterInfo.CephCred.Username` is `client.rookoperator` but here we only need `rookoperator`

Reply:
Ok, to avoid hard coding "rookoperator" here, we may want to remove the `client.` prefix.